ra/ra.go

```diff
@@ -707,7 +704,7 @@ func (ra *RegistrationAuthorityImpl) checkAuthorizationsCAA(
 	if len(badNames) > 0 {
 		return berrors.UnauthorizedError(
 			"authorizations for these names not found or expired: %s",
-			strings.Join(badNames, ", "),
+			strings.Join(core.UniqueLowerNames(badNames), ", "),
```

Comment:
I think by this point, the names have been uniqued and lowered; any reason to do it again here?

Reply:
Sorting the badNames made the corresponding unit test easier to write along side the others that did exact error matching.

Reply:
Reading it again I don't think its actually necessary. I'll try removing and seeing if the test starts to flake.

Reply:
I removed this as part of 0f50f25fb88741824870201af85e5a1f0f8fcc69 without seeming to break anything :+1: